graspologic/nominate/VNviaSGM.py

```diff
@@ -0,0 +1,387 @@
+import numpy as np
+from ..match import GraphMatch as GMP
+from sklearn.base import BaseEstimator
+import itertools
+import warnings
+
+
+class VNviaSGM(BaseEstimator):
+    """
+    This class implements Vertex Nomination via Seeded Graph Matching
+    (VNviaSGM) with the algorithm described in [1].
+
+    VNviaSGM is a nomination algorithm, so instead of completely matching two
+    graphs `A` and `B`, it proposes a nomination list of potential matches in
+    graph `B` to a vertex of interest (voi) in graph `A` with associated
```

Comment:
```suggestion
    graph `B` to a vertex of interest (VOI) in graph `A` with associated
```
abbreviation, so should be capped in text